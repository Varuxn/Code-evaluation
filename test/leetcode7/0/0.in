7
368 64 654 611 278 273 44
