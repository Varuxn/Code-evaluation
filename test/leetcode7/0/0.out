304 20 43 333 5 229 44 